PiwikTracker/PiwikUserDefaults.swift

```diff
@@ -5,8 +5,11 @@ import Foundation
 /// All getter and setter are sideeffect free and automatically syncronize
 /// after writing.
 internal struct PiwikUserDefaults {
-    static var standard = PiwikUserDefaults()
-    let userDefaults = UserDefaults.standard
+    let userDefaults: UserDefaults
+
+    init(suiteName: String?) {
```

Comment:
Maybe non optional?

I always to to use non-optimal lets unless there is a strong reason.